Tangem/Modules/Main/ActionButtons/Models/ActionButtonsTokenSelectorItem.swift

```diff
@@ -8,13 +8,20 @@
 
 import BlockchainSdk
 
-struct ActionButtonsTokenSelectorItem: Identifiable, Equatable {
+struct ActionButtonsTokenSelectorItem: Identifiable {
     let id: String
-    let tokenIconInfo: TokenIconInfo
-    let name: String
-    let symbol: String
-    let balance: LoadableTokenBalanceView.State
-    let fiatBalance: LoadableTokenBalanceView.State
     let isDisabled: Bool
+    let tokenIconInfo: TokenIconInfo
+    let infoProvider: DefaultTokenItemInfoProvider
     let walletModel: WalletModel
 }
+
+extension ActionButtonsTokenSelectorItem: Equatable {
+    static func == (lhs: ActionButtonsTokenSelectorItem, rhs: ActionButtonsTokenSelectorItem) -> Bool {
+        lhs.id == rhs.id
+            && lhs.isDisabled == rhs.isDisabled
+            && lhs.tokenIconInfo == rhs.tokenIconInfo
+            && lhs.infoProvider.id == rhs.infoProvider.id
+            && lhs.walletModel.id == rhs.walletModel.id
```

Comment:
```suggestion
            && lhs.infoProvider == rhs.infoProvider
            && lhs.walletModel == rhs.walletModel
```
В WalletModel+.swift и в DefaultTokenItemInfoProvider вроде уже есть все нужные конформансы Equatable